src/collection/style.js

```diff
@@ -71,6 +71,25 @@ var elesfn = ({
     }
   },
 
+  // get the associated values at the specified index when parsing a property with a multiple tag
+  parsedStyleMult: function( property, index ){
+    var ele = this[0];
```

Comment:
- This is too expensive to run for reading properties.
- It's potentially mutating the style for the element.
- It removes other needed info by only giving you the `value`.  You often need the `pfValue` and/or the `units` as well.

You can't have a general function like this for style properties.  

You should define a general `util.arrayValueAt( array, index, fallbackIndex )` function in `util` and use `util.arrayValueAt( ele.pstyle('foo').value, i, 0 )`.